charts/consul/Chart.yaml

```diff
@@ -22,7 +22,7 @@ annotations:
     - name: consul-dataplane
       image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.4-dev
     - name: envoy
-      image: envoyproxy/envoy:v1.26.2
+      image: envoyproxy/envoy:v1.25.11
```

Comment:
This is an annotation that gets added to the Consul Helm chart on the artifacthub.io page. It should match the default Envoy version for the legacy `apiGateway.imageEnvoy` field.

Reply:
I'm surprised this is meant to match `apiGateway.imageEnvoy`, as I'd heard otherwise and also intuitively thought it was meant to match the current version in use by default (provided by `consul-dataplane`), as with the other annos. Thanks a bunch for digging this up.

Since the intended use seems to be representing the deprecated API Gateway Envoy version, can we also add a comment here to indicate that so it's not updated incorrectly in the future? I can also update the runbook.

Reply:
Discussed offline; to minimize change for these bumps we can add this to the runbook instead or follow up later w/ a comment if needed.